plugins/multiconnect.h

```diff
@@ -0,0 +1,12 @@
+#ifndef LIGHTNING_PLUGINS_MULTICONNECT_H
+#define LIGHTNING_PLUGINS_MULTICONNECT_H
+#include "config.h"
```

Comment:
Hmm libplugin.h already includes this ?

Reply:
`config.h` is included in nearly every other header file.

Reply:
Yes, it's checked in check-source, too!